handler.py

```diff
@@ -2,46 +2,42 @@
 from calculations.calculations import Calculations
 from plots.make_plot import make_plot
 import traceback
+import pandas as pd
 
-conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv")
+conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv", "data\\output2_analog_2023-10-14_13-18-15.csv")
 calculations = Calculations("data\\output2_linpot_2023-10-14_13-23-28.csv")
+#conversions.clean_data(conversions.data)
 
 
 class handler:
 
     def wheel_load_handler(self):
         # loads and prepares initial data
-        conversions.convert_voltage_to_in()
-        conversions.convert_time(conversions.data)
-        conversions.clean_data()
-        calculations.data = conversions.data
-
-        # calculating constants
-        calculations.calculate_wheel_rate_front()
-        calculations.calculate_wheel_rate_rear()
-        calculations.calculate_wheel_rate_roll_front()
-        calculations.calculate_wheel_rate_roll_rear()
-        calculations.calculate_K_H_K_P()
-        calculations.calculate_K_R_K_W()
-        calculations.calculate_a_b()
-        calculations.calculate_q_w()
-
+        data_linpot = conversions.data_linpot
+        data_accel = conversions.data_accel
+        data_linpot_converted = conversions.convert_voltage_to_in(data_linpot)
+        data_accel_converted = conversions.convert_xl_g(data_accel)
+        df_cleaned = conversions.clean_data(data_linpot_converted)
+        calculations.data = df_cleaned
+        calculations.data2 = conversions.data_accel
+        calculations.calculate_wheel_load_constants()
+        
         # calculates wheel load formulas in pounds of force
-        calculations.calculate_force_front_left()
-        calculations.calculate_force_front_right()
-        calculations.calculate_force_rear_left()
-        calculations.calculate_force_rear_right()
+        wheel_loads = calculations.calculate_wheel_loads(df_cleaned)
 
-        plots = make_plot(calculations.data)
-        # plots.plot_wheel_load_vs_acceleration()
-        plots.plot_wheel_load_vs_time()
+        conversions.convert_time(wheel_loads)
+        conversions.convert_time(data_accel_converted)
+        plots_wheel_load = make_plot(wheel_loads)
+        plots_accel = make_plot(data_accel_converted)
+        plots_wheel_load.plot_wheel_load_vs_time(wheel_loads)
+        plots_accel.plot_acceleration_vs_time(data_accel_converted)
```

Comment:
no need for this call, only call that needs to be made is for wheel load vs time